test/test_helper.rb

```diff
@@ -35,6 +38,7 @@
 
   # for coverage we need to load all ruby files
   Dir["#{SRC_PATH}/modules/**/*.rb"].each { |f| require_relative f }
+  Dir.chdir(libdir) { Dir["**/*.rb"].each { |f| require f } }
```

Comment:
Same as above, the changes to this file are not needed. I will remove it from the PR.